otherlibs/unix/unix.ml

```diff
@@ -497,8 +497,10 @@ type tm =
     tm_yday : int;
     tm_isdst : bool }
 
-external time : unit -> float = "unix_time"
-external gettimeofday : unit -> float = "unix_gettimeofday"
+external time : unit -> (float [@unboxed]) =
+  "unix_time" "unix_time_unboxed" [@@noalloc]
+external gettimeofday : unit -> (float [@unboxed]) =
+  "unix_gettimeofday" "unix_gettimeofday_unboxed" [@@noalloc]
```

Comment:
This needs doing in `win32unix` as well

Reply:
(`win32unix/gettimeofday.c` needs equivalent boxing - `time.c` is shared)

Reply:
Done. (Let's see if appveyor agrees)